RocketChat/Rocket.Chat.ReactNative
# [FIX] Touch/Face ID toggle does not enable automatically the first time

## Proposed changes


## Issue(s)	


Jira: [TC-112]

## How to test or reproduce


- Using a device that is possible to use touch id/ face id, go to Screen Lock View, enable unlock with passcode
- Then, after save the passcode, the app will ask if you want to enable the face ID
- Test the 2 behaviors:
- If enable the face id/touch id test if you activate properly, if the switch will change to enable
- Else if you don't want to activate the face id/touch id, the switch should keep as false

## Screenshots

## Types of changes




## Checklist



## Further comments



[TC-112]: https://rocketchat.atlassian.net/browse/TC-112?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
